release_notes_generator/action_inputs.py

```diff
@@ -115,6 +118,11 @@ def validate_inputs():
         if not isinstance(skip_release_notes_label, str) or not skip_release_notes_label.strip():
             raise ValueError("Skip release notes label must be a non-empty string.")
 
+        verbose = ActionInputs.get_verbose()
+        if not isinstance(verbose, bool):
+            raise ValueError("Verbose logging must be a boolean.")
```

Comment:
I believe that all these checks should be written as:
`logger.error("message", exc_info=True)
 exit(1)`

Reply:
This was addressed by minor refactoring in commit 344de319dd2911a5ac50f4014b1c2742f1510cc7.

Reply:
It looks better with this approach!